docs/tutorials/02_speed.ipynb

```diff
@@ -0,0 +1,1074 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    em.setup(X, y, cross_validator=KFold(n_splits=3))
</pre><p>same here setup output does not contain fold output, so better to hide this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/autoemulate/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>